addons/website_slides/models/slide_channel.py

```diff
@@ -610,8 +610,8 @@ def _resequence_slides(self, slide, force_category=False):
         index_of_added_slide = ids_to_resequence.index(slide.id)
         next_category_id = None
         if self.slide_category_ids:
-            force_category_id = force_category.id if force_category else slide.category_id.id
-            index_of_category = self.slide_category_ids.ids.index(force_category_id) if force_category_id else None
+            category_id = force_category or slide.category_id.id
```

Comment:
It is the inverse, force_category is intended to be a browse. Who is calling it wiht an id ?

Reply:
I found it in 2 places:
1. in addons/website_slides/controllers/main.py
2. in  addons/website_slides/tests/test_slide_utils.py ; I change it to send the id like the controller